cezo_fl/util/model_helpers.py

```diff
@@ -73,3 +77,14 @@ def get_trainable_model_parameters(
     for param in model.parameters():
         if param.requires_grad:
             yield param
+
+
+def model_forward(
```

Comment:
This is not sufficient for the generation model now unfortunately. We need a class or something equivalently to hold the generation parameters.

Reply:
@BichengYing, it is better to support generation tasks in FedAvg.